Plugins/NativeGallery/NativeGallery.cs

```diff
@@ -855,7 +855,9 @@ public static VideoProperties GetVideoProperties( string videoPath )
 					height = 0;
 				if( !long.TryParse( properties[2].Trim(), out duration ) )
 					duration = 0L;
-				if( !float.TryParse( properties[3].Trim(), out rotation ) )
+				var rotStr = properties[3].Trim();
+				rotStr = rotStr.Replace(',', '.');
+				if ( !Single.TryParse(rotStr, NumberStyles.Float, CultureInfo.InvariantCulture, out rotation ) )
```

Comment:
Wouldn't it be simpler:

`if( !float.TryParse( properties[3].Trim().Replace( ',', '.' ), NumberStyles.Float, CultureInfo.InvariantCulture, out rotation ) )`

Reply:
@bakwc I could merge this request if you simplify the code as suggested.